app/javascript/stylesheets/sitewide/color-library.scss

```diff
@@ -39,25 +39,33 @@ $Olivine: #85BE79;
 
 
 $Wrinkle-White: #d9d9d9;
-$Eerie-Black: #191919;
+$Eerie-Black: #270C1D;
 $Faded-Bone:#e3dac9;
 $Bone:#F9F6EE;
 $Blood:#C70039;
 $Meat:#581845;
 $Organ: #5c1009;
 
+
+$Canopy: #335f5d;
+$Observatory: #04946c;
+$Parchment: #EFEADC;
+$Peachy-White: #FAF1EB;
+$Blush: #F5C2C8;
+$Watermelonade: #E84F5C;
+$Kirsch: #B11680;
+
 //Semantic Colors Names
```

Comment:
These ARENT SEMANTIC, PAT
They are priority, lets make a list  of semantic tokens and map these to them